devops/vaults.env

```diff
@@ -8,10 +8,10 @@ VUE_APP_ADDRESS_COMPLETE_KEY="op://canadapost/$APP_ENV/address-key/ADDRESS_COMPL
 VUE_APP_REGISTRY_HOME_URL="op://web-url/$APP_ENV/registry/REGISTRY_HOME_URL"
 VUE_APP_AUTH_WEB_URL="op://web-url/$APP_ENV/auth-web/AUTH_WEB_URL"
 VUE_APP_BUSINESSES_URL="op://web-url/$APP_ENV/business/BUSINESSES_URL"
-VUE_APP_DASHBOARD_URL="op://web-url/$APP_ENV/business/DASHBOARD_URL"
+VUE_APP_DASHBOARD_URL="op://web-url/$APP_ENV/business-le/DASHBOARD_URL"
 VUE_APP_SITEMINDER_LOGOUT_URL="op://web-url/$APP_ENV/siteminder/SITEMINDER_LOGOUT_URL"
-VUE_APP_BUSINESS_CREATE_URL="op://web-url/$APP_ENV/business-create/BUSINESS_CREATE_URL"
-VUE_APP_BUSINESS_EDIT_URL="op://web-url/$APP_ENV/business-edit/BUSINESS_EDIT_URL"
+VUE_APP_BUSINESS_CREATE_URL="op://web-url/$APP_ENV/business-create-le/BUSINESS_CREATE_URL"
+VUE_APP_BUSINESS_EDIT_URL="op://web-url/$APP_ENV/business-edit-le/BUSINESS_EDIT_URL"
```

Comment:
Is this change for eventual deployments using CD scripts? I think you don't need to do this for the "gcbrun" (cloudbuild) environments.

Actually, what are your environments?

Reply:
Dev for sure and I am in the process of pushing legal api in GCP to test.  So i'm thinking we will have legal name UIs running in test too so others can test.

Reply:
An actual Dev environmemnt (using a CD script), or a cloudbuilt environment ("gcbrun")?

Reply:
Isn't it that the actual Dev environment grabs the values from 1pass while the cloud build environment grabs from the `.env.example`? I was thinking to update both to tackle both scenarios.

Reply:
^^ Yes, you're right.

👍 to tackling both scenarios, but I'm still curious about your environments :)

Reply:
I'm still talking to Patrick about this.  Ideally there is just one set of environments but we are still waiting until everyone can get on legal name branch as you know.

We'll probably try to hold on in just using dev for legal name UIs for now.

Reply:
Are you actually in "dev"? Or a cloudbuild environment?

Reply:
We're actually in Dev. We haven't been using the cloudbuild (gcbrun) URLs Sev.